Usiqee/Config/Config.swift

```diff
@@ -7,11 +7,17 @@
 
 
 enum Config {
-    static var AppStoreLink = "itms-apps://itunes.apple.com/app/id1561023361"
+    static let AppStoreLink = "itms-apps://itunes.apple.com/app/id1561023361"
     
     #if DEBUG
-        static var WormholyIsEnabled = true
+        static let WormholyIsEnabled = true
     #else
-        static var WormholyIsEnabled = false
+        static let WormholyIsEnabled = false
+    #endif
+    
+    #if DEBUG
+        static let adUnitId: String = "ca-app-pub-3940256099942544/3986624511"
+    #else
+        static let adUnitId: String = "ca-app-pub-3940256099942544/3986624511"
```

Comment:
i guess you can reuse the same `#if Debug`

Reply:
Do there is a testing and production key ? on only production ?

Reply:
I only set the testing `ad unit id`. the production id wasnt provided